app/models/repository_rows_my_module.rb

```diff
@@ -0,0 +1,7 @@
+class RepositoryRowsMyModule < ActiveRecord::Base
```

Comment:
I think this class needs to be in singularity, and maybe use some additional checks (see how I've done it in my PR) - https://github.com/Ducz0r/scinote-web/blob/lm-sci-1279/app/models/repository_row_my_module.rb